### PR TITLE
Add missing VB breaking change doc

### PR DESCRIPTION
**Pull request by Youssef1313**
Related to https://github.com/dotnet/roslyn/issues/45158

### CODE REVIEW

File: docs/compilers/Visual Basic/Single-line if with End as statement.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Single-line if with `End` keyword as statement
+
+As reported in [#45158](https://github.com/dotnet/roslyn/issues/45158), it was possible to use something like the following until Visual Studio version (TODO: UNKNOWN):
```
Comment (Youssef1313) [inline docs/compilers/Visual Basic/Single-line if with End as statement.md:3]: I've only tried the snippet on Visual Studio 2015 and Visual Studio 2019. So, I don't know if this breaking change was introduced in 2017 or 2019.

**Comment by Youssef1313** [comment]
@RikkiGibson A test fails here while not editing any code files. Is this issue known?

**Comment by CyrusNajmabadi** [comment]
Was this an international breaking change?  Or the introduction of a bug?  If the latter, we should just fix the bug.

**Comment by Youssef1313** [comment]
@CyrusNajmabadi I certainly don't know if it's intentional or not. But I believe if this was intentional it would have been documented. So, it looks like a bug. But even it's a bug, it still needs to be documented IMO, and the documentation should state the version that this gets fixed in.

**Comment by CyrusNajmabadi** [comment]
I don't think we generally doc bugs. We usually have hundreds to thousands of them. The issue/pr itself is the docs for them.

**Comment by Youssef1313** [comment]
I see a bug that was considered as a breaking change is doc'ed here:

https://github.com/dotnet/roslyn/blob/b4c6fe0bea95772be7e48a2c943de700b78e78b1/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20post%20VS2017.md



> - Visual Studio 2017 15.0-15.5 shipped with a bug around definite assignment of local functions that did not produce definite assignment errors when an uncalled local function contains a nested lambda which captures a variable. For example:
>
>    ```
>    void Method()
>    {
>        void Local()
>        {
>            Action a = () =>
>            {
>                int x;
>                x++; // No error in 15.0 - 15.5
>            };
>        }
>    }
>    ```
>
>    This is changed in 15.6 to now produce an error that the variable is not definitely assigned.

**Comment by 333fred** [comment]
I would agree with Cyrus. This wasn't an intentional breaking change. This was a bug.

**Comment by Youssef1313** [comment]
Ok I'm closing this

**Pull request closed without merging**